### ISSUE TITLE
Rails/HttpPositionalArguments autocorrects existing keyword arguments incorrectly

### ISSUE THREAD

**Issue by nickcampbell18** [bbatsov/rubocop#5930]
## Expected behavior

Given the following snippet:

```ruby
RSpec.describe "GET /foo" do
  it "renders a response" do
    get "/foo", headers: {}
  end
end
```

I would not expect any offenses - this code is correct and compatible with the Rails 5 HTTP syntax.

## Actual behavior

The cop `Rails/HttpPositionalArguments` detects an offense:

```
dummy_spec.rb:3:5: C: [Corrected] Rails/HttpPositionalArguments: Use keyword arguments instead of positional arguments for http call: get.
    get "/foo", headers: {}
    ^^^
```

When we apply autocorrect, the code is incorrectly rewritten like this:

```ruby
RSpec.describe "GET /foo" do
  it "renders a response" do
    get "/foo", params: { headers: {} }
  end
end
```

(I would expect `:headers` to be ignored since it looks like it's [already in the keyword arguments list](https://github.com/bbatsov/rubocop/blob/54b71f18226121c2660cccca06b4e2a8a46ceb85/lib/rubocop/cop/rails/http_positional_arguments.rb#L25))

## Steps to reproduce the problem

The snippets above should suffice, but let me know if you need any more information!

## RuboCop version

```
$ rubocop -V
0.56.0 (using Parser 2.5.1.0, running on ruby 2.5.1 x86_64-darwin16)
```

**Comment by koic** [comment]
Thanks for opening issue.
The edge of RuboCop fixed this issue, I think that it will be solved with next release of new RuboCop (> 0.56.0) .

Additional information, the following is PR related to this Issue.
#5893

**Comment by nickcampbell18** [comment]
Thanks! Gonna close this one then.
